### PR TITLE
fix: :sparkles: new get user by id endpoint

### PR DESCRIPTION
**Pull request by adelinavisan97**

### CODE REVIEW

File: src/modules/users/users.route.ts [hunk 1]
```diff
@@ -69,6 +69,21 @@ router.get('/info', verifyMiddleware.verifyToken, async (req, res) => {
   }
 });
 
+// This endpoint will return a user's information by their ID
+router.get('/info/:id', verifyMiddleware.verifyToken, async (req, res) => {
```
Comment (WilliamWood01) [inline src/modules/users/users.route.ts:73]: should already have the user id saved as a local variable no?

Comment (adelinavisan97) [reply src/modules/users/users.route.ts:73]: yeah, but this is to get a doctor's name from the prescription (when we display the prescription details), so we need to pass the specific id. for the logged in user, it's fine, we are using the existing endpoints

Comment (WilliamWood01) [reply src/modules/users/users.route.ts:73]: ah never mind you need all of the user details?

Comment (adelinavisan97) [reply src/modules/users/users.route.ts:73]: not really, just the first and last name, but i was being lazy

Comment (WilliamWood01) [reply src/modules/users/users.route.ts:73]: okay clear makes sense

**Review by WilliamWood01 — APPROVED** [review]

**Pull request merged**